velox/functions/sparksql/coverage/Coverage.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gflags/gflags.h>
+#include <fstream>
+#include <iostream>
+#include "velox/exec/Aggregate.h"
+#include "velox/functions/CoverageUtil.h"
+#include "velox/functions/sparksql/Register.h"
+#include "velox/functions/sparksql/aggregates/Register.h"
+
+DEFINE_bool(all, false, "Generate coverage map for all Spark functions");
+
+using namespace facebook::velox;
+
+namespace {
+
+void printVeloxFunctions() {
```

Comment:
What's the difference between this function and the one used in Presto's coverage?

Reply:
Presto has a [kLinkBlockList](https://github.com/facebookincubator/velox/pull/3927/files#diff-6ab510aea06f78c8c4854b0d83bd7dafc70b20f651a41355a598456a2cbdaff8L291). And it is not useful for spark. May I can let kLinkBlockList as a arguments of printVeloxFunctions.

Reply:
A block list can be an argument and Spark can pass an empty list.

Reply:
Thanks, just fixed it.